src/fitnesseMain/FitNesseMain.java

```diff
@@ -49,6 +49,32 @@ public static void main(String[] args) {
   protected static void exit(int exitCode) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 'method def modifier' has incorrect indentation level 2, expected level should be 4. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck)